### DISCUSSION TITLE
没注释和api文档，着实理解有点困难。。对于我这种菜鸟来说。。。

### DISCUSSION THREAD

**Discussion by strayer-ckey** [ejoy/ant#188] [General]
大佬们。。

**Comment by strayer-ckey** [comment]
debug中调用c/c++的数据也看不到。。大佬们有办法吗

**Reply by cloudwu** [reply]
一般情况下，调试器看 Lua 中的数据更重要。

**Reply by actboy168** [reply]
c++调试器和lua调试器，你可以同时开

**Comment by cloudwu** [comment]
我的建议是：尝试写点小游戏，看遇到什么困难。然后在这里讨论具体的问题，根据讨论的结果，改进引擎接口或补充针对性的文档。

文档不太可能突然就有了，只能是一点点补充的。
